src/persist-client/src/paths.rs

```diff
@@ -0,0 +1,75 @@
+// Copyright Materialize, Inc. and contributors. All rights reserved.
+//
+// Use of this software is governed by the Business Source License
+// included in the LICENSE file.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0.
+
+use differential_dataflow::trace::Description;
+use mz_persist_types::Codec64;
+use uuid::Uuid;
+
+use crate::write::WriterId;
+use crate::Id;
+
+// structure of the data in s3 (all of these with some common prefix)
+//
+// - `<CollectionId>/versions/<UUID>` -> CollectionMeta
+// - `<CollectionId>/writers/<WriterId>` -> WriterMeta
```

Comment:
correct me if I'm wrong but we're not actually storing writermeta and readermeta separately anymore right?